lamindb_setup/core/_clone.py

```diff
@@ -90,4 +102,116 @@ def connect_local_sqlite(instance: str) -> None:
 
     isettings = load_instance_settings(settings_file)
     isettings._persist(write_to_disk=False)
-    isettings._load_db()
+    isettings._load_db(sqlite_read_only=read_only)
```

Comment:
the locking log appers due to this in `_load_db` https://github.com/laminlabs/lamindb-setup/blob/881a219a1a99d21bb22efa17f9198067168212c7/lamindb_setup/core/_settings_instance.py#L617